src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -137,6 +137,7 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('templating_twig.php');
         }
 
+        $container->setParameter('security.enable_authenticator_manager', $this->authenticatorManagerEnabled);
```

Comment:
I don't find a better way to "store" the information once fetched from the config file parsing

Reply:
I prefer doing `$container->getDefinition('data_collector.security')->addArgument($this->authenticatorManagerEnabled);` instead of exposing this parameter to everyone.

Reply:
Thanks It works with `addArgument` :tada: 
I tried with `setArgument()` without success, I guess you should respect the parameters order with `addArgument`? Or it can be resolved with Type hint ?

Reply:
Yes, `addArgument()` adds arguments after the already defined arguments (from the config).